main.ts

```diff
@@ -217,16 +217,32 @@ class Translator {
         break;
 
       case ts.SyntaxKind.VariableDeclarationList:
+        // Note that a VariableDeclarationList can only occur as part of a for loop.
         var varDeclList = <ts.VariableDeclarationList>node;
-        this.visitList(varDeclList.declarations, ';');
+        this.visitList(varDeclList.declarations, ',');
```

Comment:
the comma is the default separator in list, so you can just not specify the second param.